src/Orleans.Runtime/Silo/Silo.cs

```diff
@@ -66,7 +66,7 @@ public enum SiloType
         private readonly IncomingMessageAgent incomingAgent;
         private readonly IncomingMessageAgent incomingSystemAgent;
         private readonly IncomingMessageAgent incomingPingAgent;
-        private readonly Logger logger;
+        private readonly ILogger logger;
```

Comment:
as a general rule of thumb, also migrated silo.cs off Logger, since we should all try to migrate in our PRs! It's a team effort